docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -154,11 +155,13 @@ Preparing For a Release
 
    Go to the `BuildBot web status`_ and force a build on the just-created branch.
 
-   In addition, review the link-check step of the documentation builder to ensure that all the errors are expected.
+   In addition, review the link-check step of the documentation builder to ensure that all the errors (the links with "[broken]") are expected.
 
 #. Make a pull request on GitHub
 
    The pull request should be for the release branch against ``master``, with a ``[FLOC-123]`` summary prefix, referring to the release issue that it resolves.
+   The pull request will not be automatically linked to the JIRA issue because it does not use the standard branch naming scheme.
+   Therefore, you will have to manually submit the release issue for review, and link to the pull request in a comment.
```

Comment:
Does putting the ticket slug in the pull request name not link the pull-request to the ticket?